test/apisix/Dockerfile.test.apisix.deb

```diff
@@ -0,0 +1,34 @@
+ARG IMAGE_BASE="ubuntu"
+ARG IMAGE_TAG="focal"
+
+FROM ${IMAGE_BASE}:${IMAGE_TAG}
+
+ARG ETCD_VERSION="v3.4.14"
+ARG APISIX_VERSION
+
+ENV RUNNING_ETCD_VERSION=${ETCD_VERSION}
+
+COPY apisix_${APISIX_VERSION}-0_amd64.deb /apisix_${APISIX_VERSION}-0_amd64.deb
+
+# install openresty
+RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y libreadline-dev lsb-release libpcre3-dev libssl-dev perl build-essential \
+    && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends wget gnupg ca-certificates \
+    && wget -O - https://openresty.org/package/pubkey.gpg | apt-key add - \
+    && echo "deb http://openresty.org/package/ubuntu $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/openresty.list \
+    && DEBIAN_FRONTEND=noninteractive apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y openresty openresty-openssl111-dev
+
+# install etcd
+RUN set -x \
+    && wget https://github.com/etcd-io/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz \
+    && tar -zxvf etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+
+# install apisix
+RUN set -x \
+    && dpkg -i /apisix_${APISIX_VERSION}-0_amd64.deb
+
+# start etcd and test
+CMD ["sh", "-c", "(nohup etcd-$RUNNING_ETCD_VERSION-linux-amd64/etcd >/tmp/etcd.log 2>&1 &) && sleep 10 && apisix init && apisix init_etcd && /usr/local/openresty/bin/openresty -p /usr/local/apisix -g 'daemon off;'"]
```

Comment:
Why not run `apisix start` directly?

Reply:
Because the container will exit when the process itself exits

Reply:
Could we use a placeholder like `sleep 3600s`?